src/meta/meta_track.c

```diff
@@ -67,6 +67,35 @@ __meta_track_next(WT_SESSION_IMPL *session, WT_META_TRACK **trkp)
 }
 
 /*
+ * __meta_track_clear --
+ *	Clear the structure.
+ */
+static void
+__meta_track_clear(WT_SESSION_IMPL *session, WT_META_TRACK *trk)
+{
+	__wt_free(session, trk->a);
+	__wt_free(session, trk->b);
+	memset(trk, 0, sizeof(WT_META_TRACK));
```

Comment:
This assumes that ```WT_ST_EMPTY``` is 0 (which it currently is). I'd rather make that explicit in the ```enum``` if the code requires it.